src/services/jwt.service.js

```diff
@@ -0,0 +1,39 @@
+import jwt from 'jsonwebtoken';
+
+/**
+ * Generate token
+ * Verify token
+ * @class JwtService
+ */
+class JwtService {
+  /**
+   *
+   *
+   * @param {data} data
+   * @returns {token} @memberof JwtService
+   */
+  static generateToken(data) {
+    const token = jwt.sign(
+      data, process.env.SECRET_KEY,
+      { expiresIn: process.env.TOKEN_EXPIRE_TIME }
+    );
+    return token;
+  }
```

Comment:
it should be good if you return jwt.sign() function instead of declaring a variable and return that variable do it like u did in jwt.verify function

Reply:
Ok